src/ScheduledTask/ProcessNotificationsHandler.php

```diff
@@ -194,4 +264,24 @@ private function markAsDone(string $notificationId)
         $this->notificationService->changeNotificationState($notificationId, 'done', true);
         $this->logger->debug("Payment notification {$notificationId} marked as done.");
     }
+
+    private function doRefund(OrderTransactionEntity $orderTransaction, string $transitionState, Context $context)
+    {
+        try {
+            if ($transitionState === OrderTransactionStates::STATE_PARTIALLY_REFUNDED) {
```

Comment:
The force is strong with this one ;)

```suggestion
            if (OrderTransactionStates::STATE_PARTIALLY_REFUNDED === $transitionState) {
```